src/objects/api/v2/openapi.yaml

```diff
@@ -88,13 +88,51 @@ paths:
         description: 'The desired ''Coordinate Reference System'' (CRS) of the response
           data. According to the GeoJSON spec, WGS84 is the default (EPSG: 4326 is
           the same as WGS84).'
+      - in: query
+        name: data_attr
+        schema:
+          type: string
+        description: |
+          Only include objects that have attributes with certain values.
+
+          A valid parameter value has the form `key__operator__value`.
+          `key` is the attribute name, `operator` is the comparison operator to be used and `value` is the attribute value.
+          Note: Values can be string, numeric, or dates (ISO format; YYYY-MM-DD).
+
+          Valid operator values are:
+          * `exact` - equal to
+          * `gt` - greater than
+          * `gte` - greater than or equal to
+          * `lt` - lower than
+          * `lte` - lower than or equal to
+          * `icontains` - case-insensitive partial match
+          * `in` - in a list of values separated by `|`
+
+          `value` may not contain double underscore or comma characters.
+          `key` may not contain comma characters and includes double underscore only if it indicates nested attributes.
+
+
+
+          Example: in order to display only objects with `height` equal to 100, query `data_attr=height__exact__100`
+          should be used. If `height` is nested inside `dimensions` attribute, query should look like
+          `data_attr=dimensions__height__exact__100`
+
+          This filter is very similar to the old `data_attrs` filter, but it has two differences:
+
+          * `value` may contain commas
+          * only one filtering expression is allowed
+
+          If you want to use several filtering expressions, just use this `data_attr` several times in the query string.
+          Example: `data_attr=height__exact__100&data_attr=naam__icontains__boom`
```

Comment:
Nitpicky, but ideally `explode: true` would be specified here to indicate that it should be used like `?data_attr=...&data_attr=...` (see https://swagger.io/docs/specification/v3_0/serialization/#query-parameters)

For example: https://github.com/open-formulieren/open-forms/blob/3a702080d35c0670588cc69b8e49445e6224989c/src/openforms/appointments/api/views.py#L64

Although looking at it, apparently `explode: true` is considered the default by OAS3, so I'm not sure if our schema is correct in that case :thinking:. Might be something to investigate in another issue